solidity/random-beacon/contracts/test/GroupsStub.sol

```diff
@@ -16,14 +16,9 @@ contract GroupsStub {
 
     function addCandidateGroup(
         bytes calldata groupPubKey,
-        uint32[] calldata members,
-        uint8[] calldata misbehavedMembersIndices
+        uint32[] calldata activeMembers
```

Comment:
This change is a subject to another PR I believe.

Reply:
agreed